lib/smack/SmackRep.cpp

```diff
@@ -76,8 +137,20 @@ string SmackRep::type(const llvm::Type* t) {
     return BOOL_TYPE;
   else if (isFloat(t))
     return FLOAT_TYPE;
-  else
+  else if (isInt(t)) {
+    if (BIT_VECTOR) {
+      stringstream s;
+      s << "i" << getIntSize(t);
+      return s.str();
```

Comment:
Construction of Boogie fixed-width integer type strings should be refactored into a separate function, isolating the string constant "i" to a single location:
````
string SmackRep::int_type(unsigned width) {
  stringstream s;
  s << "i" << width;
  return s.str();
}
````